lib/ngx/ssl.md

```diff
@@ -369,6 +369,11 @@ Sets the SSL certificate chain opaque pointer returned by the
 
 Returns `true` on success, or a `nil` value and a string describing the error otherwise.
 
+Note that the `set_cert` function will perform slightly better, in terms of CPU cycles wasted, than then
+[set_der_cert](#set_der_cert) variant, since the first function uses opaque cdata pointers
+which do not require any additional conversion needed to be performed by the NGINX core during the SSL session
```

Comment:
Yes, you are absolutely right. Actually, my first commit at bb1302ebcf3bc6e1543eb587c0ac3b819a392d83 
mentioned that the conversion is being made in OpenSSL, but I thought it was too detailed. 

Anyway, I suggest the following change :

` .... since the first function uses opaque cdata pointers which do not require any 
additional conversion needed to be performed by the NGINX core during the SSL 
session negotiation.`

will be replaced with :

` .... since the first function uses opaque cdata pointers which do not require any 
additional conversion to be performed by OpenSSL, while the NGINX core tries to
complete the session negotiation.`

What do you think ?